sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -25,6 +25,18 @@ export function useFormioDropdownLoader() {
     }));
   };
 
+  /**
+   * find the formio field and set the value
+   * @param form form.
+   * @param fieldId field api/id from formio.
+   * @param value value that need to be set to the field
+   */
+  const loadText = async (form: any, fieldId: string, value: any) => {
+    // load the value to the formio fields.
+    const fieldObject = formioUtils.getComponent(form, fieldId);
```

Comment:
I believe that the formio Utils has a helper to set the value of a component. Can you please check and use it instead?